.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+# .github/workflows/release.yaml
+
+on:
+  release:
+    types:
+    - created
+name: Build Release
+jobs:
+  release-assests:
+    name: release linux/amd64
+    runs-on: ubuntu-latest

```

Comment:
```suggestion
    runs-on: ubuntu-18.04

```

Reply:
We didn't pursue the latest because we met servery problems when the GitHub Action environment updates.